__tests__/plugins/network.test.ts

```diff
@@ -149,114 +154,34 @@ describe('network', () => {
 
     const delayTime = 100;
     server.route('/chunked', async (req, res) => {
+      await delay(delayTime);
       res.writeHead(200, {
         'content-type': 'application/javascript',
       });
       res.write('a');
       await delay(delayTime);
-      res.write('b');
-      await delay(delayTime);
-      return res.end('c');
+      return res.end('b');
     });
     server.route('/index', async (_, res) => {
       res.setHeader('content-type', 'text/html');
       res.end(`<script src=${server.PREFIX}/chunked />`);
     });
 
-    await driver.page.goto(server.PREFIX + '/index');
-    await driver.page.waitForLoadState();
-    await Gatherer.stop();
+    await driver.page.goto(server.PREFIX + '/index', {
+      waitUntil: 'networkidle',
+    });
     const netinfo = await network.stop();
+    await Gatherer.stop();
     expect(netinfo.length).toBe(2);
     expect(netinfo[1]).toMatchObject({
       url: `${server.PREFIX}/chunked`,
       status: 200,
       response: expect.any(Object),
       timings: expect.any(Object),
     });
-    expect(netinfo[1].timings.total).toBeGreaterThan(delayTime);
-  });
-
-  describe('waterfall timing calculation', () => {
```

Comment:
Is there a good reason to remove this? It seems helpful if for no other reason than to ensure that the data we send to heartbeat/uptime is consistent. Is that checked elsewhere?

Reply:
We don't do calculation in the Synthetics code anymore, It's as part of Playwright code.